README.md

```diff
@@ -1,8 +1,56 @@
-# Tsunami Events Dashboard (Python) 
-This app will contain a landing page with three tiles; an interactive geographical map that users can pan across to see the location of each tsunami as well as its strength, a time series graph showing the number of deaths by country, and a table presenting the top 10 strongest tsunamis. All three graphics will be filterable by using the collapsible menu which contains two widgets; a slider to select a range of years and a dropdown menu to filter which countries they would like to see. The geographical map makes use of the latitude and longitude in the data set to identify locations where a tsunami occurred and will have a heat map to represent the magnitude of the tsunami as well as the ability for users to hover over a tsunami to glean more information about a particular tsunami. Similarly, users will also be able to use this feature in the table of strongest tsunamis which will have a dropdown to select the top 5, top 10, or top 20 strongest tsunamis of the chosen period. Lastly, the time series graph will show the number of deaths by country and is filterable by a period and countries.
+# Tsunami Events Dashboard (Python)
+
+## Accessing the App via Heroku
+
+Link to Heroku app: [tsunami-events-dashboard](https://dashboard.heroku.com/apps/tsunami-events-dashboard)
+
+## Description of the App Interface
+
+This app contains a landing page with three tiles: an interactive geographical map that users can pan across to see the location of each tsunami as well as its strength, a time series graph showing the number of deaths by country, and a table listing the strongest tsunamis. The tsunami events data underlying the three plots is filtered for using a collapsible menu that contains two widgets: a slider to select a range of years of occurrence, and a drop-down menu to filter for countries impacted. The geographical map makes use of the tsunami latitude and longitude location data, generates a heat map to indicate tsunami magnitude, and allows users to hover over tsunami events plotted on the map to glean more comprehensive event details. Users can also peruse of a table listing the strongest tsunami events per the year and country selection applied, with the option to select from among a display of the top 5, 10 , 20 strongest events. Lastly, the time series graph shows the number of deaths by country per the year and country selection applied.
 
 ## Proposal
-Our proposal can be found via this link: [proposal](proposal.md)
+
+Our proposal can be found via this link: [proposal](docs/proposal.md)
 
 ## Dashboard Sketch
+
 ![dashboard sketch](img/tsunami_sketch.jpg)
+
+## Accessing the App Locally
+
+To run and explore the app locally, first clone the git repo, create and activate a local environment, and install dependencies:
+
+    git clone https://github.com/UBC-MDS/tsunami-events-dashboard-python.git
+
+    # mac
+    virtualenv venv
```

Comment:
we have yet to create 'venv'. Also, I believe it should be conda activate `venv`. Please correct me if I am wrong